crates/re_types_builder/src/codegen/python/views.rs

```diff
@@ -109,6 +110,12 @@ See [rerun.blueprint.archetypes.SpaceViewContents][]."
 Defaults to true if not specified."
                 .to_owned(),
         ),
+        (
+            "defaults",
+            "List of default components or component batches to add to the space view. When an archetype
+in the view is missing a component included in this set, the value of default will be used
+instead of the normal fallback for the visualizer.".to_owned(),
```

Comment:
are we handling batches for default correctly already? The way things are set up I expect it to just work but did you get around to testing that?
E.g. setting several colors for color default